example-nextjs/components/Navbar.tsx

```diff
@@ -65,15 +68,16 @@ export default function Navbar({ current }) {
                       }
                     >
                       {mounted && resolvedTheme === 'dark' ? (
-                        <SunIcon></SunIcon>
+                        <SunIcon />
                       ) : (
-                        <MoonIcon></MoonIcon>
+                        <MoonIcon />
                       )}
                     </button>
                   </div>
                 </div>
                 <div className="flex items-center -mr-2 gap-1 sm:hidden">
                   <button
+                    aria-label={`Activate ${resolvedTheme} mode`}
```

Comment:
I think this should use the same logic that you did on 60-62:

```typescript
                      aria-label={`Activate ${
                        resolvedTheme === 'dark' ? 'light' : 'dark'
                      } mode`}
```

Reply:
Correct Thanks fixed now